rtengine/CMakeLists.txt

```diff
@@ -176,6 +176,16 @@ if(LENSFUN_HAS_LOAD_DIRECTORY)
     set_source_files_properties(rtlensfun.cc PROPERTIES COMPILE_DEFINITIONS RT_LENSFUN_HAS_LOAD_DIRECTORY)
 endif()
 
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "12.0")
```

Comment:
Thanks for the patch. Perhaps it's worthwhile to add a comment referring to the issue? Or else explain why this code is here.

Reply:
Added a comment